src/xcode/ENA/ENA/Source/Scenes/TraceLocations/TraceLocationsCoordinator.swift

```diff
@@ -30,9 +30,7 @@ class TraceLocationsCoordinator {
 		
 		#if DEBUG
 		if isUITesting {
-			if let TraceLocationsInfoScreenShown = UserDefaults.standard.string(forKey: "TraceLocationsInfoScreenShown") {
-				store.traceLocationsInfoScreenShown = (TraceLocationsInfoScreenShown != "NO")
-			}
+			store.traceLocationsInfoScreenShown = UserDefaults.standard.bool(forKey: UITestingLaunchArguments.infoScreen.traceLocationsInfoScreenShown.remove(prefix: "-"))
```

Comment:
And another thing: When now using real bool, can't we use true/ false and not longer YES/NO? That's so ObjC....